.github/workflows/ci.yml

```diff
@@ -292,6 +292,9 @@ jobs:
           # See also https://github.com/linebender/vello/pull/610
           SKIP_RENDER_TESTS: ${{ matrix.skip_gpu }}
 
+      - name: Run cargo nextest on unsafe arena
+        run: cargo nextest run -p tree_arena --locked --no-fail-fast --no-default-features
```

Comment:
Ideally we'd use miri here, I think?

Reply:
We run Miri in the `color` CI so you can take a peek there.